src/showcase/src/pages/displayManage.astro

```diff
@@ -6,10 +6,13 @@ import Screen from "../layouts/Screen.astro";
   <script>
     import { toast } from "$src/components/toast";
     import { userNumber } from "../constants";
-    import { identityBackground } from "../backgrounds";
     import { dapps } from "../constants";
     import { displayManagePage } from "$src/flows/manage";
     import { html } from "lit-html";
+    import identityCardBackground from "$src/assets/identityCardBackground.png";
```

Comment:
Importing it here we ensure that it doesn't get mixed with the application code and we don't need to cast types.

Ideally, we will move the showcase to its own library with its own package.json and then we will be able to separate import statements better.